docs/pipeline-components-and-applications/iglu/iglu-repositories/iglu-server/reference/index.md

```diff
@@ -6,13 +6,27 @@ sidebar_position: 0
 
 This is a complete list of the options that can be configured in the Iglu Server HOCON config file. The [example configs in github](https://github.com/snowplow-incubator/iglu-server/tree/master/config) show how to prepare an input file.
 
+## License
+
+Iglu Server is released under the [Snowplow Limited Use License](https://docs.snowplow.io/limited-use-license-1.0/) ([FAQ](/docs/contributing/limited-use-license-faq/index.md)).
+
+To accept the terms of license and run Iglu Server, set the `ACCEPT_LIMITED_USE_LICENSE=yes` environment variable. Alternatively, you can configure the `license.accept` option, like this:
```

Comment:
```suggestion
To accept the terms of license and run Iglu Server, set the `ACCEPT_LIMITED_USE_LICENSE=true` environment variable. Alternatively, you can configure the `license.accept` option, like this:
```

Should this be `true` or `yes`?  The config uses a `boolean` value.

Reply:
Either works. `true`, `yes`, `on`, `1`, case insensitive. Don’t ask why :D